src/Services/NumberFormatter.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Services;
+
+class NumberFormatter
+{
+    private const MILLION = 999500;
+    private const NEGATIVE_MILLION = -999500;
+    private const ONE_HUNDRED_K = 99950;
+    private const NEGATIVE_ONE_HUNDRED_K = -99950;
+    private const THOUSAND = 999.9999;
+    private const NEGATIVE_THOUSAND = -1000;
+    private const ZERO = 0;
+    private const MILLION_SIGN = 'M';
+    private const THOUSAND_SIGN = 'K';
+
+
+    /**
+     * @param float $number
+     * @return string
+     */
+    public function formatNumbers(float $number): string
+    {
+        $result = 0;
+
+        switch ($number) {
```

Comment:
`switch` veikia ne taip.

Reikėtų naudoti `if` arba

```php
switch (true) {
```

Reply:
Čia geriau naudoti `if`us arba kažką pan į:
```php
swtich (true) { // Įvykdys, kai mųsšų sąlyga bus lygi true
 case ($absValue >= 999500): // atitinka: if (($absValue >= 999500) == true)
   ///
 breal
 case ($absValue >= 1000): // atitinka: if (($absValue >= 1000) == true)
   ///
  break;
}
```
```php
swtich (false) { // Įvykdys, kai mųsšų sąlyga bus lygi true
case ($absValue >= 999500): // atitinka: if (($absValue >= 999500) === false)
  /// Priešinga sąlyga
breal
case ($absValue >= 1000): // atitinka: if (($absValue >= 1000) === false)
  /// Priešinga sąlyga
 break;
}
```

Pačiau:
 * https://www.php.net/manual/en/control-structures.switch.php

Reply:
Supratau, pataisysiu. Tiesiog buvau radęs, kad su `switch`'ais visas procesas vyksta žymiai greičiau negu su if, dėl to ir nusprendžiau daryti su jais.